app.py

```diff
@@ -61,17 +61,15 @@ def registration():
 # login page to verify if a user exists
 # NOTE user cannot be verified until DB is set up and conditional statements
 # will have to be updated
-# NOTE Had trouble routing user to profile page in App.js once user is validated
 @app.route("/login", methods=["GET", "POST"])
 def login():
     username = flask.request.form.get("username")
 
     if username:
-        return flask.redirect(flask.url_for("index"))
+        return flask.render_template("profile.html")
```

Comment:
Since this is a `render_template` call instead of a redirect, the URL will still read "/login" while the profile page is shown. "/profile" should definitely be its own route.